katib-ui-rock/rockcraft.yaml

```diff
@@ -19,36 +19,86 @@ platforms:
   amd64:
 
 parts:
-  katib-ui:
+  frontend-lib:
+    plugin: nil
+    source: https://github.com/kubeflow/kubeflow
+    source-type: git
+    # Set source-tag here to the commit cited in https://github.com/kubeflow/katib/blob/master/pkg/ui/v1beta1/frontend/COMMIT
+    # for this Katib release
+    # Note: Katib 0.16.0 cannot build with Kubeflow v1.8-branch because Kubeflow refactored this lib
+    source-commit: 046c6d3c8301a95a2809c0d927bbbf72638dd639
+    # Omit source-depth because we're cloning a commit
+    # source-depth: 1
+    build-environment:
+      # suppress analytics collection
+      - NG_CLI_ANALYTICS: false
```

Comment:
I remember not having this creating some prompts during `npm ci` (I think), but do we need this?

Reply:
I honestly dont know :)  I see it in some but not all of our rocks, and some dockerfiles, which makes me think it isn't actually necessary?

Reply:
I don't mind removing, I just don't really know the consequences.  I do think if we omit it, the analytics default to `true`, which I think means our built app might be sending telemetry to someone which I think we don't want

Reply:
Yep they might be sent to Angular project. Good point actually, let's keep it then